internal/auth0/connection/expand.go

```diff
@@ -1078,3 +1078,40 @@ func checkForUnmanagedConfigurationSecrets(configFromTF, configFromAPI map[strin
 
 	return warnings
 }
+
+func expandSCIMConfigurationMapping(data *schema.ResourceData) *[]management.SCIMConfigurationMapping {
+	mapping := []management.SCIMConfigurationMapping{}
```

Comment:
It doesn't seem like we can ever get this as nil if we instantiate it like this. Is that intended?